### ISSUE TITLE
Use Computed Answer

### ISSUE THREAD

**Issue by DE0CH** [CP-STA/STAOJ#34]
Sometimes, a problem can have multiple correct answers and need to be checked with computation. A naïve example is outputing a list of number if any order. For some, you can ask the contestant to order it a certain way to make sure the answer is exactly the same, but sometimes it's impossible such as [codeforces 1719B](https://codeforces.com/contest/1719/problem/B). 

Therefore I propose to write a js file named `comparator.js` with one exported function named `compareAnswer(user, judge)` in a problem's folder which returns `true` iff the answer is right. Because `judge` is just a string, it should be quite flexible. The judge string can just contain information about the problem that helps the function to judge the answer. 

What do you guys think @Daniel5055 @kay-ventris1?

**Comment by Daniel5055** [comment]
Yeah I think this is a good idea. Shouldn't be too hard to check if a problem contains a comparator file and then import dynamically, else use a default compare answer which compares from testCase files. I was actually writing that default function now. But what does the `user` arg refer to?

**Comment by kay-liechtenstein** [comment]
Oh actually, problems I have written so far are designed in such a way that the answer for each testcase in each problem is unique. If allowing multiple correct answers is feasible it'll be appreciated, but I have not experienced hardship because that was restricted.

ps) Sorry for the delay in response.

**Comment by DE0CH** [comment]
> I was actually writing that default function now. But what does the `user` arg refer to? 

The string of user's answer.
